main/src/ca/uwaterloo/flix/language/phase/Namer.scala

```diff
@@ -689,7 +689,7 @@ object Namer {
       // Introduce a rigid region variable for the region.
       val regionVar = Symbol.freshUnkindedTypeVarSym(Ast.VarText.SourceText(sym.text), isRegion = true, loc)
 
-      mapN(visitExp(exp, ns0)) {
+      mapN(visitExp(exp, ns0)(level.incr, flix)) {
```

Comment:
Copy comment from elsewhere.

Reply:
We introduce variables both in Namer and Kinder?

Reply:
And resolver?

Reply:
Yes :)